packages/InteractiveProfilingTool-Core.package/IPTReport.class/instance/initializeWithUserCallsRootsNode..st

```diff
@@ -0,0 +1,6 @@
+initialization
+initializeWithUserCallsRootsNode: aNode
```

Comment:
Shouldn't this be called `initializeWithUserCallsRootNode`? (the change roots to root)

Reply:
s. o. bzw. #52